.env.example

```diff
@@ -0,0 +1,11 @@
+SECRET="Replace with secret string"
+
+DEV_DB_HOST=dbHost
```

Comment:
Comments have been removed! `dotenv` apparently doesn't have any support for comments. We just need to update the `README.md` to reflect the changes.